hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestDatanodeManager.java

```diff
@@ -1126,4 +1127,33 @@ public MockDfsNetworkTopology(){
       super();
     }
   }
+
+  @Test
+  public void testComputeReconstructedTaskNum() {
+    int xmitsInProgress = 13000000;
```

Comment:
One minor suggestion. We should better add an UT that `totalReplicateBlocks * maxTransfers` overflow been fixed here, because it is the reason causes xmitsInProgress be very large.  Others LGTM.

Reply:
Thank you for your valuable suggestion! I'll make the required code changes promptly.